### ISSUE TITLE
norm_grads

### ISSUE THREAD

**Issue by myway0101** [jakeret/tf_unet#141]
Thank you for sharing this awesome Unet implementation.

I have a simple question about 'norm_grads' in Unet.py code.

:param norm_grads: (optional) true if normalized gradients should be added to the summaries

I read the above description in the code. However, I don't understand what is the role of norm_grads.

Could you explain this? Is it an important parameter for training and related to performance?

**Comment by jakeret** [comment]
If true `tf_unet` computes the normalized gradients during training an makes them available in the tensorboard. This is merely for debugging purpose as it is relatively computational costly

**Comment by myway0101** [comment]
Thank you for your quick response! 
Here, debugging means to see the training loss in tensorboard?

**Comment by jakeret** [comment]
When you open the tensorboard you see many more statistics reported including the loss and the normalized gradients

**Comment by myway0101** [comment]
Thank you very much :)
